### DISCUSSION TITLE
奖励额度相关问题

### DISCUSSION THREAD

**Discussion by TechnologyStar** [BurnHair/docs#157] [分享讨论]
对于奖励额度，2024年4月8日之前邀请、且账号状态正常的用户，每人计 $5；4月8日-6月25日，每人计 $1

那之后每邀请一人可以得多少奖励额度?

**Comment by BennyThink** [comment]
现在是0 因为有很多人自己邀请自己薅羊毛。出台缓解措施之后，会恢复
